dask_ml/model_selection/_incremental.py

```
@@ -935,17 +936,17 @@ def __init__(
         scoring=None,
         verbose=False,
         prefix="",
-        scores_per_fit=None,
+        **kwargs,
```

We don't want to have kwargs here. Too easy to silently ignore typos.